src/node_crypto.cc

```diff
@@ -5107,12 +5107,13 @@ void DiffieHellman::SetKey(const v8::FunctionCallbackInfo<v8::Value>& args,
       BN_bin2bn(reinterpret_cast<unsigned char*>(Buffer::Data(args[0])),
                 Buffer::Length(args[0]), nullptr);
   CHECK_NE(num, nullptr);
-  set_field(dh->dh, num);
+  USE(set_field(dh->dh, num));
```

Comment:
Since you're threading through the return value, you might as well `CHECK_EQ(1, set_field(dh->dh, num));`.

Reply:
Sounds good, I'll add the check. Thanks